tests/test_service.py

```diff
@@ -84,7 +86,30 @@ def __iter__(self):
     rp_service.collect_tests(mocked_session)
     rp_service.start_pytest_item(mocked_item)
 
-    expect(mocked_item_start.call_count == 1, 'One HTTP POST sent')
+    expect(mocked_item_start.call_count == 2, 'Two HTTP POST sent')
```

Comment:
Why did u change it?

Reply:
When I mock mocked_module type (mock.create_autospec(Module)), call count changed to 2.
So, now I return call_count to 1, and create separated fixtures to fix it